modules/RTC/JitsiRemoteTrack.js

```diff
@@ -132,6 +199,17 @@ export default class JitsiRemoteTrack extends JitsiTrack {
         this.rtc.eventEmitter.emit(RTCEvents.REMOTE_TRACK_UNMUTE, this);
     }
 
+    /**
+     * Removes attached event listeners and dispose TrackStreamingStatus .
+     *
+     * @returns {Promise}
+     */
+    dispose() {
+        this.disposeTrackStreamingStatus();
```

Comment:
We are checking if `this._trackStreamingStatusImpl` is defined in `disposeTrackStreamingStatus` which is only possible when FeatureFlags.isSourceNameSignalingEnabled is true. However, that could change and it would be good practice to also add it here